README.md

```diff
@@ -338,6 +338,8 @@ API | Description | Auth | HTTPS | CORS |
 ### Dictionaries
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
+| [Chinese Character Web API](http://ccdb.hemiola.com/) | Chinese character definitions and pronunciations | No | No | No |
```

Comment:
Title should not end with "... API". Every entry is an API here!

```suggestion
| [Chinese Character Web](http://ccdb.hemiola.com/) | Chinese character definitions and pronunciations | No | No | No |
```

Can you fix this, please?

Reply:
Fixed!